src/com/palmergames/bukkit/towny/object/Nation.java

```diff
@@ -43,28 +43,27 @@
 	private Town capital;
 	private double taxes, spawnCost;
 	private boolean neutral = false;
-	private String nationBoard = "/nation set board [msg]", tag;
+	private String nationBoard = "/nation set board [msg]";
+	private String tag = "";
 	public UUID uuid;
 	private long registered;
 	private Location nationSpawn;
-	private boolean isPublic;
-	private boolean isOpen;
+	private boolean isPublic = TownySettings.getNationDefaultPublic();
+	private boolean isOpen = TownySettings.getNationDefaultOpen();
+	private transient List<Invite> receivedinvites = new ArrayList<>();
+	private transient List<Invite> sentinvites = new ArrayList<>();
+	private transient List<Invite> sentallyinvites = new ArrayList<>();
 
 	public Nation(String name) {
-
-		setName(name);
-		tag = "";
-        isPublic = TownySettings.getNationDefaultPublic();
-        isOpen = TownySettings.getNationDefaultOpen();
+		super(name);
 	}
 
 	public void setTag(String text) throws TownyException {
 
-		if (text.length() > 4)
+		if (text.length() > 4) {
```

Comment:
I thought we weren't doing this anymore.

Reply:
This was before you told me not to do any formatting changes anymore, literally in the middle of changing it you told me that